src/_actions/assetTypes.actions.js

```diff
@@ -15,4 +22,31 @@ export const loadAssetTypes = () => (dispatch) => {
     }));
 };
 
-export default loadAssetTypes;
+export const createAssetType = newAssetType =>
+  dispatch => axios.post('asset-types', newAssetType).then((response) => {
+    dispatch(createAssetTypeSuccess(response.data));
+    dispatch(updateToastMessageContent('Asset Type Saved Successfully',
+      'success'));
+  }).catch((error) => {
+    dispatch(createAssetTypeFailure(error));
+    dispatch(updateToastMessageContent('Could Not Save The Asset Type',
+      'error'));
+  });
+
+export const createAssetTypeSuccess = assetType => (
+  { type: CREATE_ASSET_TYPE_SUCCESS, payload: assetType }
+);
+
+export const createAssetTypeFailure = error => (
+  { type: CREATE_ASSET_TYPE_FAILURE, payload: error }
+);
+
+export const updateToastMessageContent = (message, type) => (
```

Comment:
Arthur centralized this in his PR, please liaise with him to know when he's merged to develop. You'll have to import `updateToastMessageContent`, not create it again

Reply:
Ok, I will liaise with him.

Reply:
@HawiCaesar this hasn't been deleted yet